roles/openstack-network/handlers/main.yml

```diff
@@ -2,3 +2,14 @@
 - name: ifup neutron external interface
   command: ifup {{ neutron_external_interface }}
   ignore_errors: yes
+
+- name: enable ip forwarding
+  sysctl: name="net.ipv4.conf.all.forwarding" value=1 sysctl_set=yes state=present reload=yes
+ 
+- name: enable ip masq
```

Comment:
This is necessary for allinone deployments as well, which is why we kept it in a general network role.